src/olympia/amo/tests/test_amo_utils.py

```diff
@@ -26,12 +25,14 @@
 u = u'Ελληνικά'
 
 
-def test_slug_validator():
-    eq_(slug_validator(u.lower()), None)
-    eq_(slug_validator('-'.join([u.lower(), u.lower()])), None)
-    assert_raises(ValidationError, slug_validator, '234.add')
-    assert_raises(ValidationError, slug_validator, 'a a a')
-    assert_raises(ValidationError, slug_validator, 'tags/')
+class TestSlugValidator(TestCase):
```

Comment:
okay, I'll change to pytest.raises.  

I'm not sure that one function is going to make a difference though when we have many hundreds of subclasses of TestCase - going through and swapping out TestCase for one of the simpler superclasses where database rollback isn't needed would be a solution but I'll leave that for someone else :)